grep.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Grep
```

Comment:
This cheatsheet are options of GNU grep, not of POSIX grep. POSIX grep has much less options (one example: it doesn't have PCRE)